packages/data-point/README.md

```diff
@@ -1946,12 +2003,12 @@ dataPoint.addEntities({
 | *after*   | [Reducer](#reducers) | reducer to be resolved **after** the entity resolution |
 | *outputType*  | String, [Reducer](#reducers) | [type checks](#entity-type-check) the entity's output value, but does not mutate it |
 | *error*   | [Reducer](#reducers) | reducer to be resolved in case of an error |
-| *params*  | `Object` | User defined Hash that will be passed to every transform within the context of the transform's execution |
+| *params*  | `Object` | user defined Hash that will be passed to every transform within the context of the transform's execution |
 
 
-#### <a name="request-entity">Request Entity</a>
+### <a name="request-entity">Request Entity</a>
 
-Requests a remote source, using [request](https://github.com/request/request) behind the scenes. The features supported by `request` are exposed/supported by Request entity.
+Requests a remote source, using [request-promise](https://github.com/request/request-promise) behind the scenes. The features supported by `request-promise` are exposed/supported by Request entity.
```

Comment:
request-promise is just a wrapper of `request`, should point back to `request`

Reply:
`request-promise` has some additional options though: https://github.com/request/request-promise#api-in-detail -- do we not want people to use them?